webpack.config.js

```diff
@@ -4,6 +4,7 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 /* eslint-enable */
```

Comment:
- [ ] I see an error in the browser console:
    ![imagen](https://user-images.githubusercontent.com/60789317/144095842-385a2b33-b7d0-4946-8f70-25f7a33a6085.png)
    - I think that here you the function that you are trying to use is `addEventListener` instead of `Listener`, kindly fix this 😄

Reply:
@helman101 I'm a little confused here, at my end I'm not getting such errors. It's is spelt as `addEventListener` and not `Listener` at my end also